File: SymulatorSwiataJava/src/Zwierzeta/Wilk.java
package Zwierzeta;
import java.awt.Color;

import Organizmy.*;

public class Wilk extends Zwierze{

	private Zasoby a;
	@Override
	public void akcja(Swiat swiat, int i, int j) {
		nowe = losuj();
		kolizja(swiat, i, j);
		wiek++;
	}
	
	public Wilk(int x1, int y1)
	{
		wiek = 0;
		x = x1;
		y = y1;
		wyglad = 'W';
		inicjatywa = 5;
		sila = 9;
		color = Color.RED;
	}

}

File: SymulatorSwiataCpp/SymulatorSwiata/Wilk.cpp
#include "Wilk.h"
#include "zasoby.h"
#include "Swiat.h"

Wilk::Wilk(int x, int y)
{
	this->wiek = 0;
	this->x = x;
	this->y = y;
	this->wyglad = WILK;
	this->inicjatywa = 5;
	this->sila = 9;
}

void Wilk::akcja(Swiat &swiat, int i, int j)
{
	nowe = losuj();
	this->kolizja(swiat, i, j);
	wiek++;
}